.internal/charsEndIndex.js

```diff
@@ -10,9 +8,14 @@ import baseIndexOf from './baseIndexOf.js'
  * @returns {number} Returns the index of the last unmatched string symbol.
  */
 function charsEndIndex(strSymbols, chrSymbols) {
-  let index = strSymbols.length
+  const chrSymbolsDict = {}
+  let index = chrSymbols.length
+  while (index--) {
+    chrSymbolsDict[chrSymbols[index]] = true
```

Comment:
flyby review: if you're doing this for performance then use a `Map` instead of an object, it's much faster

Reply:
@Toxicable Look at performance comparison: https://jsperf.com/set-vs-obj-comparison/1

Reply:
sure set works also :P

Reply:
> sure set works also :P

I used an object instead of Set(or Map) because Set is not implemented in old browsers. And for me it's not clear that in this case Set(or Map) will be a better way.

Reply:
> sure set works also :P

So what should I do?